tests/test_module.py

```diff
@@ -102,50 +102,50 @@ def test_exports():
         },
     ]
 
-def test_imports():
```

Comment:
Because we have a bug in Wasmer. Will fix it. That's temporary.

Reply:
Let's open an issue on this repo after merging the PR, so we don't forget :)